vue-components/stories/Button.stories.ts

```diff
@@ -34,3 +35,27 @@ export function primary(): Component {
 		`,
 	};
 }
+
+export function iconOnly(): Component {
```

Comment:
At some point, we should create separate pages for iconOnly buttons per variant ([see structure](https://docs.google.com/document/d/1loENZ13z5s_Nes4exm5Z9d5S3PnR7F_0brv5azQ5XB4/edit)). This could logically be a DS constellation task 🙋‍♀️